ch.sbb.scion.rcp.microfrontend/src/ch/sbb/scion/rcp/microfrontend/browser/JxBrowserView.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Project: RCS - Rail Control System
+ *
+ * © Copyright by SBB AG, Alle Rechte vorbehalten
+ */
+package ch.sbb.scion.rcp.microfrontend.browser;
+
+import java.util.function.Consumer;
+
+import org.eclipse.swt.widgets.Display;
+
+import com.teamdev.jxbrowser.browser.Browser;
+import com.teamdev.jxbrowser.js.JsFunctionCallback;
+import com.teamdev.jxbrowser.js.JsObject;
+import com.teamdev.jxbrowser.navigation.event.FrameLoadFinished;
+import com.teamdev.jxbrowser.view.swt.BrowserView;
+
+/**
+ * JxBrowser implementation of the {@link BrowserView} interface.
+ */
+public class JxBrowserView extends AbstractBrowserView {
+
+  private final Browser browser;
+  private final BrowserView browserView;
+
+  public JxBrowserView(final BrowserView browserView) {
+    this.browser = browserView.getBrowser();
+    this.browserView = browserView;
+
+    browser.navigation().on(FrameLoadFinished.class, e -> {
+      if (e.frame().isMain()) {
+        notifyFrameLoadFinished();
+      }
+    });
+  }
+
+  @Override
+  public void loadUrl(final String url) {
+    browser.navigation().loadUrlAndWait(url);
+  }
+
+  @Override
+  public Object executeJavaScript(final String javaScript) {
+    return browser.mainFrame().orElseThrow().executeJavaScript(javaScript);
+  }
+
+  @Override
+  public boolean isFocused() {
+    return browserView.isFocusControl();
+  }
+
+  @Override
+  public DisposableJsFunction registerJsFunction(final String name, final boolean once, final Consumer<Object[]> callback) {
+    JsObject window = browserView.getBrowser().mainFrame().orElseThrow().executeJavaScript("window");
+    // Define the JsFunctionCallback to handle invocations from JavaScript
```

Comment:
Ideally,
```java
// Define the `JsFunctionCallback` to handle invocations from JavaScript.
```
but at the very least:
```java
// Define the JsFunctionCallback to handle invocations from JavaScript.
```

Please review the rest of use cases accordingly.